tests/mysql/model/test_model.py

```diff
@@ -316,7 +316,7 @@ def test_attribute_check_with_hasattr(self):
         self.assertFalse(hasattr(Profile(), "__password__"))
 
 
-if os.getenv("RUN_MYSQL_DATABASE", None).lower() == "true":
+if os.getenv("RUN_MYSQL_DATABASE", "false").lower() == "true":
```

Comment:
_(my testing environment wasn't set up, so this caused an error, saying that `lower` is not a function on `NoneType`)_